src/btree/bt_handle.c

```diff
@@ -477,7 +477,7 @@ __btree_conf(WT_SESSION_IMPL *session, WT_CKPT *ckpt)
 
     /* Set special flags for the history store table. */
     if (strcmp(session->dhandle->name, WT_HS_URI) == 0) {
-        F_SET(btree, WT_BTREE_HS);
+        F_SET(btree->dhandle, WT_DHANDLE_HS);
```

Comment:
This is more of a question to @sulabhM or whoever the other reviewer will be. But do we think that this is still the right spot to set this flag considering that it's no longer a `btree` flag?

Reply:
I think it is.